dashboard/db/migrate/20170222192220_remove_href_from_external.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveHrefFromExternal < ActiveRecord::Migration[5.0]
+  def up
+    External.all.each{|level| level.update!(properties: level.properties.except('href'))}
+  end
```

Comment:
It is irreversible.

Is the noop down migration just to make it explicit that this is the case? What would happen if I were to run migrate:down without a noop down migration? (I would have guessed it would be the same behavior)